app/src/main/res/layout/the_institute_view_fragment.xml

```diff
@@ -0,0 +1,213 @@
+                        android:background="@drawable/blue_circle_background" />
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="10dp"
+                        android:fontFamily="@font/roboto_light"
+                        android:text="@string/the_institute_p4"
+                        android:textAlignment="textStart"
+                        android:textColor="#000062"
+                        android:textSize="18sp" />
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="30dp"
+                    android:layout_marginEnd="30dp">
+
+                    <FrameLayout
+                        android:layout_width="15dp"
+                        android:layout_height="15dp"
+                        android:layout_marginTop="5dp"
+                        android:background="@drawable/blue_circle_background" />
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="10dp"
+                        android:fontFamily="@font/roboto_light"
+                        android:text="@string/the_institute_p5"
+                        android:textAlignment="textStart"
+                        android:textColor="#000062"
+                        android:textSize="18sp" />
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="30dp"
+                    android:layout_marginEnd="30dp">
+
+                    <FrameLayout
+                        android:layout_width="15dp"
+                        android:layout_height="15dp"
+                        android:layout_marginTop="5dp"
+                        android:background="@drawable/blue_circle_background" />
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="10dp"
+                        android:fontFamily="@font/roboto_light"
+                        android:text="@string/the_institute_p6"
+                        android:textAlignment="textStart"
+                        android:textColor="#000062"
+                        android:textSize="18sp" />
+                </LinearLayout>
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="20dp"
```

Comment:
@sitamadex11  I would recommend that we should extract the size from dimen.xml if exit and create a one and export . It's good practice to do so.....

Reply:
Yeah thats really a good practice. Though there has many range of dimensions we have used. but still if you want to work on it can raise a issue.